README.md

```diff
@@ -71,15 +72,17 @@ In addition to the Generalized Linear Models described above, Photon-ML also sup
 ### GAME - Generalized Additive Mixed Effect Model
 GAME is a specific expansion of traditional Generalized Linear Models that further provides entity level (e.g., per-user/per-item) or segment level (e.g., per-country/per-category) coefficients, also known as random effects in the statistics literature, in addition to global coefficients. It manages to scale model training up to hundreds of billions of coefficients while still solvable within Spark's framework.
 
-GAME models consist of three components:
-  * One fixed effect model:
-    * The fixed effect model is effectively a conventional generalized linear model. Its parameters are "global" in the sense that they apply uniformly to all entities.
-  * Multiple random effect models:
-    * Random effect models consist of "local" parameters – entity-specific coefficients that can be seen as random deviations from the global mean. In other words, they are personalized models.
-  * Optionally a matrix factorization model:
-    * The matrix factorization model captures interactions between the different random effect models.
+Currently Photon-ML supports GAME models consist of the following three types of components:
+  * Fixed effect model:
+    * Each fixed effect model is effectively a conventional generalized linear model. Its parameters are "global" in the sense that they apply uniformly to all entities.
+  * Random effect model:
+    * Each random effect model consists of "local" parameters – entity-specific coefficients that can be seen as random deviations from the global mean, for example, a per-user random effect model that models each user's behavior through user-specific coefficients in additional to the fixed-effect coefficients shared by all users.
```

Comment:
"... from the global mean. For example, a per-user random effect models each user's behavior through user-specific coefficients."

I can't think of a way to add the statement about 'in addition to shared coefficients' without it sounding like a random effect model computes both. I think leaving the above + the links to the tutorial/presentation/paper should be quite clear.